homeassistant/components/sensor/__init__.py

```diff
@@ -390,7 +392,7 @@ class SensorDeviceClass(StrEnum):
     VOLTAGE = "voltage"
     """Voltage.
 
-    Unit of measurement: `V`
+    Unit of measurement: `V`, `mv`
```

Comment:
Please ensure that you also update NumberDeviceClass accordingly
```suggestion
    Unit of measurement: `V`, `mV`
```